ts/store/actions/organizations.ts

```diff
@@ -4,14 +4,14 @@
 import { ActionType, createStandardAction } from "typesafe-actions";
 import { ServicePublic } from "../../../definitions/backend/ServicePublic";
 
-export const setSelectedOrganizations = createStandardAction(
-  "SET_SELECTED_ORGANIZATIONS"
+export const deleteUselessOrganizations = createStandardAction(
+  "DELETE_USELESS_ORGANIZATIONS"
```

Comment:
if the previous is changed, it could be also `removeOrganizations` similar to `removeServiceTuples`